misc/generate_test_files.sh

```diff
@@ -0,0 +1,75 @@
+#!/usr/bin/env bash
+#
+# Dependencies:
+#   - psql, on Ubuntu: 'sudo apt install postgresql-client-12'
+#   - Start the SDK test environment to setup a DB at localhost:5433
+#
+# Arguments (modes)
+#
+# data [query] - print all encoded transactions, useful to quickly find which
+#                TXIDs are useful for whatever test is being written.
+#                Optionally provide a query.
+
+export PGPASSWORD='harness'
+
+TEMP_FILE='/tmp/copy_tmp'
+
+
+# Arg1 TXID        txid to download using 'escape' notation, the leading slash will be added back in.
```

Comment:
such a TXID would be fragile though? What source data set is the txid from? That's likely to change, right?

Reply:
It's from the SDK test environment, so they should not change. This script happens to be using "dataset2" (the snapshot you emailed me recently), if/when I need to pull transactions from another dataset I'll turn that into a parameter.

Since the data being extracted should never change, including them in this script is mainly to serve as a reference for how the previous files were generated.